torch/_inductor/pattern_matcher.py

```diff
@@ -342,17 +342,29 @@ def __repr__(self):
         return f"{self.__class__.__name__}({', '.join(args)})"
 
     def _match(self, node: torch.fx.Node, ctx: MatchContext):
-        if (
-            not self._match_fns(node)
-            or len(node.args) != len(self.args)
-            or len(node.kwargs) != len(self.kwargs)
-        ):
+        if not self._match_fns(node):
             return FailedMatch(f"function_mismatch: node={node}, pattern={self}")
 
         if not self._match_users(node, ctx):
             return FailedMatch(f"multiple_users {node}")
 
-        node_items, node_spec = self.flatten(node.args, node.kwargs)
+        _args = node.args
+        _kwargs = node.kwargs
+        if len(_args) != len(self.args) or len(_kwargs) != len(self.kwargs):
+            from torch.fx.operator_schemas import normalize_function
+
+            arg_types = [type(node.args[i]) for i in range(len(node.args))]
```

Comment:
i'm not sure how this is working, since `arg_types` here will give you `torch.fx.node.Node`